servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -317,6 +327,7 @@ public static final class Builder {
         static final Duration DEFAULT_EWMA_HALF_LIFE = Duration.ofSeconds(10);
         static final long DEFAULT_CANCEL_PENALTY = 5L;
         static final long DEFAULT_ERROR_PENALTY = 10L;
+        private boolean cancellationIsError;
```

Comment:
I think it's worth considering cancellation to be errors as the default since timeouts will be a very common reason for cancellation. Opinions welcome.

Reply:
I did end up changing the default because we already consider cancellation as a half-penalty for the P2C algorithm. Opinions appreciated.